src/Illuminate/Http/Client/Concerns/DeterminesStatusCode.php

```diff
@@ -146,15 +146,25 @@ public function conflict()
     }
 
     /**
-     * Determine if the response was a 422 "Unprocessable Entity" response.
+     * Determine if the response was a 422 "Unprocessable Content" response.
      *
      * @return bool
      */
-    public function unprocessableEntity()
+    public function unprocessableContent()
     {
         return $this->status() === 422;
     }
 
+    /**
+     * Determine if the response was a 422 "Unprocessable Content" response.
+     *
+     * @return bool
```

Comment:
Maybe add a `@deprecated` annotation here.

Reply:
Was thinking about the same